requirements.md

```diff
@@ -0,0 +1,117 @@
+# Requirements for Open Screen Protocol
+
+This document outlines functional and non-functional requirements of the Open
+Screen protocol.
+
+Functional requirements derive from the two Web APIs that the protocol will support:
+- The [Presentation API](https://w3c.github.io/presentation-api/)
+- The [Remote Playback API](https://w3c.github.io/remote-playback/)
+
+Non-functional requirements address efficiency, user experience, and security
+aspects of the protocols.
+
+## Presentation API
+
+The Presentation API allows one browser (the *controlling user agent*, or
+*controller*) to discover and initiate presentation of Web content on a second
+user agent (the *receiving user agent*, or *receiver*).  We use the term
+*display* to refer the entire device responsible for implementing the
+*receiver*, including browser, OS, networking and display.
```

Comment:
Done.

I can check the definition of _presentation display_ in the spec when I take on https://github.com/w3c/presentation-api/issues/345.